app/api/uploadURL.ts

```diff
@@ -0,0 +1,32 @@
+import { Storage } from '@google-cloud/storage'
+import { BlitzApiHandler } from 'blitz'
+
+import {
+  gcpStorageBucketName,
+  gcpStorageClientEmail,
+  gcpStoragePrivateKey,
+  gcpStorageProjectId,
+} from 'app/core/environment'
+
+const gc = new Storage({
+  projectId: gcpStorageProjectId,
+  credentials: {
+    client_email: gcpStorageClientEmail,
+    private_key: gcpStoragePrivateKey,
+  },
+})
```

Comment:
have you initialized this in `gcpStorage.ts`?

Reply:
This one is deprecated, but I didn't delete it because the other files in `api` folder seem to be the same and they weren't deleted 👀